src/logging.ts

```diff
@@ -40,7 +40,12 @@ let logLevel = DEFAULT_LOG_LEVEL;
 
 export function log(level: LogLevel, ...message: unknown[]) {
   if (logLevelValues[level] <= logLevelValues[logLevel]) {
-    console.log(`[${level}] Bootstrapper ${message}`);
+    const messageStr = message
+      .map(m => {
+        return typeof m === 'object' ? JSON.stringify(m) : m;
+      })
+      .join(' ');
+    console.log(`[${level}] Bootstrapper ${messageStr}`);
```

Comment:
I wanted to support in the logger passing objects, previously 

this log call:
`log(LogLevel.INFO, 'Platform: ', platformInfo);`

would output as:

`[INFO] Bootstrapper Platform: ,[object Object]`

but now looks like:

`[INFO] Bootstrapper Platform:  {"os":"darwin","arch":"arm64"}`

Reply:
Sounds good, but (NP) we can simplify this code since `console.log` is already able to parse/print objects without stringifying them

```suggestion
    console.log(`[${level}] Bootstrapper`, ...message);
```

To me this prints
`[INFO] Bootstrapper Platform:  { os: 'linux', arch: 'x64' }`

Reply:
yeah, simpler is better. I will have a look at what was the original issue for the `Object object` output